articles/vmware/vmw-how-vls-add-products-4.md

```diff
@@ -0,0 +1,265 @@
+
+3. Enter the basic user information and click **Next**.
+
+4. To assign role and allocate resources, in the _Assign Groups and Permissions_ window, click **Objects**.
+
+5. From the _Select Role_ drop-down menu, select **Administrator** and select the Assign this role to the user check box.
+
+6. In the _Select Object Hierarchies_ pane, select the **vSphere Storage** check box.
+
+7. The vSphere inventory tree appears in the Select Object pane.
+
+8. In the Select Object pane, select the virtual machines to be metered and click **Finish**.
+
+9. Go to the vCloud Usage Meter Web interface and add or update the user credentials for vRealize Operations Manager to use the new account.
+
+### Adding Horizon to the Usage Meter
+
+Customers can meter Horizon deployments starting with Horizon 7.10 and up.
+
+> [!NOTE]
+> Horizon metering does not include NSX for Desktop or VSAN for Desktop metering, these must be added separately to usage meter using the NSX or vSAN metering.
+
+Horizon usage is based on the highest concurrent connections per month. Usage Meter will track the calendar month statistics and will reset its count at the beginning of a new month. This supports both standard Horizon desktops and Horizon Apps usage.
+
+If a Horizon Connection Server exists behind a load balancer, you must add a Connection Server driectly to vCloud Usage Meter, not via a load balanced IP.
+
+If multiple Connection servers are deployed in a load balanced cluster, you must add only one of the Connection Servers to vCloud Usage Meter.
+
+1. In the main menu bar of the vCloud Usage Meter Web interface, click **Products**.
+
+2. In the left navigation bar, click **Horizon**.
+
+3. On the _Horizon_ page, click **Add**.
+
+4. In the **Endpoint** field, enter the host name or IP address of your VMware Horizon instance.
+
+5. In the **Username** and **Password** fields, enter the credentials for the VMware Horizon instance.
+
+6. Enter the VMware Horizon domain name.
+
+7. Click **Add**.
+
+## Checking the monitoring status
+
+Once you have finished adding all of your metered products you can check monitoring status to ensure usage meter is monitoring products correctly
+
+1. In the main menu bar of the vCloud Usage Meter Web interface, click **Notifications**.
+
+2. Using the Time range fields you can change the dates to display.
+
+3. The top area will display an overall status of each days collections. Each day is represented by an icon displaying the overall collection status.
+
+4. The bottom area displays the current months worth of collections and can be filtered using the columns or by clicking on the date in the top section
+
+5. If collections are failing, hovering the mouse over the icon in the top section, will display how many of errors were recorded and clicking on the icon will filter the bottom window and display the error messages.
+
+Check Below steps
```

Comment:
I'm assuming that this was a note to yourself to check that this step was accurate... Can you confirm that the step is good?

Reply:
Removed note